go/database/mpt/io/exportable.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package io
+
+import (
+	"github.com/Fantom-foundation/Carmen/go/common"
+	"github.com/Fantom-foundation/Carmen/go/database/mpt"
+)
+
+type exportable interface {
```

Comment:
I do not think this interface name is very well fitting. Having a look at the interface name and its method, what suggests  the methods` Visit() `+ getters compose something exportable? Exportable interface I would normally expect methods like `export`, `convert`, `send`, etc. 

I would say this interface - judging from its methods - is `TrieVisitor`, or even more generic `StateVisitor`, or back more specific `MptStateVisitor`. Wdyf?

Reply:
Also the interface should be documented